config/ses.toml

```diff
@@ -1,6 +1,24 @@
-[default]
+[ses]
 
-# SES version (can be 6 or 7)
-# The configured node image must match the SES supported distro. For example,
-# SLES 15 SP1 for SES6, and SLES 15 SP2 for SES7.
-ses_version = 7
+# The target version of SES to install. This must match a key below that
+# contains the repositories and yaml substitutions.
+
+ses_target = "ses7"
+
+  [ses.ses7]
+    [ses.ses7.yaml_substitutions]
+      "suse.com" = "suse.de/suse/containers/ses/7/containers"
+
+    [ses.ses7.repositories]
+      # Each key/value under this sub-heading is a repository
+      storage = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
+      storage_updates = "http://download.suse.de/ibs/SUSE/Updates/Storage/7/x86_64/update/"
+
+  [ses.devel]
+    [ses.devel.yaml_substitutions]
+      "suse.com" = "suse.de/suse/containers/ses/7/containers"
+
+    [ses.devel.repositories]
+      # Each key/value under this sub-heading is a repository
+      storage = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
```

Comment:
these are not devel repos

Reply:
No. It's the repo to test the latest milestone

Reply:
This is the set of repos used in the legacy CI. What set should we be validating in the devel pipeline?